modules/admin-portal/pages/gsql-output-file.adoc

```diff
@@ -1,36 +1,38 @@
 = Download GSQL Output File
 :sectnums:
 
-In a GSQL query, users can print any output from the query to a file. You can download the query output files from Admin Portal. 
+In a GSQL query, you can print any output from the query to a local file on the machine running TigerGraph. (See xref:gsql-ref:querying:output-statements-and-file-objects.adoc[] for details.)
+You can download these files from your web browser through the Admin Portal interface.
 
+This also applies to GraphStudio, because GraphStudio does not allow files to be saved from the web browser directly.
+Instead, any files saved through a query on GraphStudio must first be saved on the server, such as in a folder of your choice under `\home\tigergraph\`, then downloaded via Admin Portal.
+
+
+[discrete]
 == Procedure
```

Comment:
I don't think adding a discrete tag here is a good idea. I think this would prevent the entire TOC from displaying